Sources/Card/ConfirmPaymentSourceBody.swift

```diff
@@ -0,0 +1,17 @@
+import UIKit
+#if canImport(PaymentsCore)
+import PaymentsCore
+#endif
+
+struct ConfirmPaymentSourceBody: APIRequestBody {
```

Comment:
Request body objects are fully encodable and can be directly set as a property of `APIRequest`.